proto/rill/admin/v1/api.proto

```diff
@@ -202,6 +202,24 @@ service AdminService {
     option (google.api.http) = {delete: "/v1/tokens/current"};
   }
 
+  // ListServiceAuthTokens lists all the service auth tokens
+  rpc ListServiceAuthTokens(ListServiceAuthTokensRequest) returns (ListServiceAuthTokensResponse) {
+    option (google.api.http) = {get: "/v1/tokens/service"};
+  }
+
+  // IssueServiceAuthToken returns the temporary token for given service account
+  rpc IssueServiceAuthToken(IssueServiceAuthTokenRequest) returns (IssueServiceAuthTokenResponse) {
+    option (google.api.http) = {
+      post: "/v1/tokens/service",
+      body: "*"
+    };
+  }
+
+  // RevokeServiceAuthToken revoke the service auth token
+  rpc RevokeServiceAuthToken(RevokeServiceAuthTokenRequest) returns (RevokeServiceAuthTokenResponse) {
+    option (google.api.http) = {delete: "/v1/tokens/service"};
+  }
```

Comment:
1. Move them to after the service-related APIs
2. Use the same paths as the services – like `/v1/organizations/{organization_name}/services/{name}/tokens`